gui/default/assets/css/overrides.css

```diff
@@ -341,6 +341,24 @@ ul.three-columns li, ul.two-columns li {
     z-index: 980;
 }
 
+/*
+ * Folder Basic Ignores Tree tweaks
+ */
+
+#folderGlobalTree-container {
```

Comment:
The styles here seem like a copy from `#restoreTree-container`. Is this correct? If yes, then I think it would be nice to just create a common class with these to add to the both elements to avoid the repetition.

Reply:
Yes, I just copied that.

I will look at this later, when I know the style is final. `min-height` and `max-height` might need some tuning.

Reply:
merged with original since I decided to not chane it.